<launch>
  <group ns="warehouse">
    <group ns="goods">
      <node pkg="robonomics_game_warehouse" name="yellow" type="warehouse_node.py">
        <param name="size" value="[3, 4]" />
        <param name="offset" value="[0, 0]" />
        <param name="warehouse" value="goods" />
        <param name="content" value="yellow" />
      </node>
      <node pkg="robonomics_game_warehouse" name="green" type="warehouse_node.py">
        <param name="size" value="[3, 4]" />
        <param name="offset" value="[3, 0]" />
        <param name="warehouse" value="goods" />
        <param name="content" value="green" />
      </node>
      <node pkg="robonomics_game_warehouse" name="blue" type="warehouse_node.py">
        <param name="size" value="[3, 4]" />
        <param name="offset" value="[6, 0]" />
        <param name="warehouse" value="goods" />
        <param name="content" value="blue" />
      </node>
      <node pkg="robonomics_game_warehouse" name="purple" type="warehouse_node.py">
        <param name="size" value="[3, 4]" />
        <param name="offset" value="[9, 0]" />
        <param name="warehouse" value="goods" />
        <param name="content" value="purple" />
      </node>
    </group>
  </group>
  <param name="opcua_client_node" value="/opcua/opcua_client" />
  <group ns="transport_goods">
    <node pkg="robonomics_game_transport" name="stacker_goods" type="stacker_node.py">
      <param name="opcua_server_namespace" value="ns=3;s=/Airalab/StackerGoods" />
      <param name="direction" value="backward" />
      <param name="timeout" value="5000" />
    </node>
    <node pkg="robonomics_game_transport" name="conveyor" type="conveyor_node.py">
      <param name="opcua_server_namespace" value="ns=3;s=/Airalab/Conveyor" />
      <param name="tr_load_time" value="2000" />
      <param name="tr_unload_time" value="2000" />
      <param name="tr_timeout" value="5000" />
      <param name="tl_unload_time" value="2000" />
    </node>
    <node pkg="robonomics_game_transport" name="sensor" type="color_sensor_node.py">
      <param name="opcua_server_namespace" value="ns=3;s=/Airalab/Sensor" />
    </node>
    <node pkg="robonomics_game_transport" name="storage" type="storage_node.py">
      <param name="stacker_node" value="/transport_goods/stacker_goods" />
    </node>
  </group>
</launch>
